src/main/java/sparksoniq/jsoniq/tuple/FlworTuple.java

```diff
@@ -104,7 +102,7 @@ public boolean isRDD(String key, IteratorMetadata metadata) {
             || dfVariables.containsKey(key);
     }
 
-    public boolean isDF(String key, IteratorMetadata metadata) {
+    public boolean isDataFrame(String key, IteratorMetadata metadata) {
```

Comment:
This can be moved to a separate PR, reviewed and merged.

Reply:
Done. Please review at #446